src/app/layout/basket-form/basket-form.tsx

```diff
@@ -213,70 +213,51 @@ const BasketForm = () => {
                 <ul role="list" className="divide-y divide-gray-200 border-b border-t border-gray-200">
                   {lineItems.map((product, productIdx) => (
                     <li key={product.id} className="flex py-6 sm:py-10 relative">
-                      <div className="absolute right-3 top-3">
+                      <div className="absolute right-8 top-8">
                         <button
                           type="button"
                           className="-m-2 inline-flex p-2 text-gray-400 hover:text-gray-500"
                           onClick={() => handleRemoveProductClick(product.id, product.quantity)}
                         >
                           <span className="sr-only">Remove</span>
-                          <XMarkIconMini className="h-5 w-5" aria-hidden="true" />
+                          <TrashIcon className="h-5 w-5" aria-hidden="true" />
                         </button>
                       </div>
-                      <div className="flex-shrink-0">
+                      <div className="relative flex-shrink-0">
                         <Link to={`/product/${product.productId}`}>
                           <img
                             src={product.variant?.images?.[0]?.url || ''}
                             alt={product.name.en}
-                            className="h-24 w-24 rounded-md object-cover object-center sm:h-48 sm:w-48"
+                            className="h-32 w-32 rounded-md object-cover object-center sm:h-48 sm:w-48"
                           />
                         </Link>
+                        {product.variant?.attributes?.find((item) => item.name === 'Danger') ? (
+                          <div className="absolute right-0 top-0 z-[1] overflow-hidden w-full h-full text-right">
+                            <div className="absolute bottom-0 left-0 bg-red-600 s text-base font-normal font-tahoma text-white text-center transform h-6 w-full rounded-b-md">
+                              Attention!
+                            </div>
+                          </div>
```

Comment:
+ it looks good
+ This can be further improved, for example like this:  
```js 
                          <div className="absolute inset-0 z-10 overflow-hidden flex items-end">
                            <p className="flex-1 bg-red-600 text-base text-white text-center rounded-b-md">
                              Attention!
                            </p>
                          </div>
```